dev-aws/kafka-shared/iam_dpd.tf

```diff
@@ -0,0 +1,50 @@
+resource "kafka_topic" "iam_dpd_v1" {
+  name               = "auth-customer.iam-dpd-v1"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # retain 100MB on each partition
+    "retention.bytes" = "104857600"
+    # keep data for 7 days
+    "retention.ms" = "604800000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+module "iam_dpd_producer" {
```

Comment:
Let's move away from the consumer & producer modules and use the tls-app instead. You can put consumers & producers under the same app with it.
I'm currently working on migrating all iam from these modules: https://github.com/utilitywarehouse/kafka-cluster-config/compare/refactor-iam-modules , but not yet ready